apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-landing.tsx

```diff
@@ -448,6 +450,10 @@ export const SignInMethodLanding: FunctionComponent<SignInMethodLandingPropsInte
                                             onClick={ () => onLoginFlowSelect(LoginFlowTypes.APPLE_LOGIN) }
                                             showSetupGuideButton={ false }
                                             showCardAction={ false }
+                                            disabledHint={ t("console:develop.pages." +
+                                                "authenticationProviderTemplate.disabledHint.apple") }
+                                            disabled={ new URL(deploymentConfig?.serverOrigin)?.
+                                                hostname === "localhost" }
```

Comment:
Shall we move "localhost" to a const file

Reply:
Fixed with ef7449959e83515d9dce295ac89af4fe12b6cbff